### ISSUE TITLE
Fail to refocus window

### ISSUE THREAD

**Issue by beefeater7** [axcore/tartube#5]
Clicking on the main window with another program in focus will not bring up TarTube if a dialog is open, only unfocus the other application. 
![unfocused](https://user-images.githubusercontent.com/10468970/60340670-beb4a080-999b-11e9-8f3c-02a8ecaf2dfb.PNG)

I believe standard behavior refocuses program with dialog blinking on top.

**Comment by axcore** [comment]
This appears to be a problem with the Gtk graphics library. I ran the example script from section 17.1.1 at https://python-gtk-3-tutorial.readthedocs.io/en/latest/dialogs.html, and I saw exactly the same effect. I tested the same script on various Linux systems, and the windows behave correctly.

I'm not sure what can be done about this. Control of dialogue windows is handled by Gtk itself. The calling code does nothing more than call Gtk.Dialogue.run(), and then wait for a response.
